<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SteamUserSelectClass</class>
 <widget class="QDialog" name="SteamUserSelectClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>464</width>
    <height>274</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string comment="Steam user selection dialog title.">Steam User</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QWidget" name="wgtSteamUsers" native="true">
     <layout class="QHBoxLayout" name="steamUserLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="wgtEdit" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>4</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLineEdit" name="edtUserId">
        <property name="maxLength">
         <number>20</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDialogButtonBox" name="bboxControls">
        <property name="standardButtons">
         <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>bboxControls</sender>
   <signal>accepted()</signal>
   <receiver>SteamUserSelectClass</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>231</x>
     <y>252</y>
    </hint>
    <hint type="destinationlabel">
     <x>231</x>
     <y>136</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bboxControls</sender>
   <signal>rejected()</signal>
   <receiver>SteamUserSelectClass</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>231</x>
     <y>252</y>
    </hint>
    <hint type="destinationlabel">
     <x>231</x>
     <y>136</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>edtUserId</sender>
   <signal>textChanged(QString)</signal>
   <receiver>SteamUserSelectClass</receiver>
   <slot>UserIdChange(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>231</x>
     <y>225</y>
    </hint>
    <hint type="destinationlabel">
     <x>231</x>
     <y>136</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>UserIdChange(QString)</slot>
 </slots>
</ui>
